src/Form.php

```diff
@@ -728,7 +728,7 @@ public function ajaxSubmit()
         $this->on('change', 'input, textarea, select', $this->js()->form('remove prompt', new JsExpression('$(this).attr("name")')));
 
         if (!$this->canLeave) {
-            $this->js(true, (new JsChain('atk.formService'))->preventFormLeave($this->name));
+            $this->js(true, (new JsChain('atk.formService'))->preventFormLeave($this->name), $this->formElement);
```

Comment:
@ibelar please also check this selector/commit https://github.com/atk4/ui/pull/1275/commits/cd33102170cd0f452aa77a9ad81183e58cfa5dee